Apps/jellyseerr/docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+name: jellyseerr
+services:
+  jellyseerr:
+    cpu_shares: 50
+    deploy:
+      resources:
+        limits:
+          memory: 256M
+    environment:
+      - LOG_LEVEL=debug
+      - TZ=$TZ
+    image: fallenbagel/jellyseerr:1.7.0
+    ports:
+      - mode: ingress
+        target: 5055
+        published: "5055"
+        protocol: tcp
+    restart: unless-stopped
+    volumes:
+      - type: bind
+        source: /DATA/AppData/$AppID
```

Comment:
This should be: `/DATA/AppData/$AppID/config`

Reply:
not sure about that bc on my casaos instance I don't have the /config in volume 🤷